cfgov/jinja2/v1/owning-a-home/prepare-worksheets/index.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+{% set active_page = "prepare-worksheets" %}
+{% set page_js = "prepare-worksheets" %}
+
+{% block title %}Owning a Home: Preparation worksheets{% endblock title %}
+
+{% block content %}
+
+<main class="prepare-worksheet content content__1-3" id="main" role="main">
+  <div class="wrap content_wrapper">
+    <aside class="content_sidebar">
+      {% with active_page = "prepare-worksheets" %}
+        {% include "sidebar_nav.html" %}
+      {% endwith %}
+    </aside>
+    <section class="content_main">
+      <section class="page-contents">
+      </section>
+      <section class="type-details">
+       
+        <nav class="block block__padded-top block__border-top">
+          <button class="btn btn__disabled btn__super btn-worksheet-prev">
```

Comment:
This file was deleted in https://github.com/cfpb/cfgov-refresh/pull/3165/commits/adea54f8ecb4f983b08882aaf842c2e812e402bf.